Cargo.toml

```diff
@@ -46,10 +46,15 @@ http = "1.2.0"
 http-body = "1.0.1"
 hyper = { version = "1.5.2", features = ["client"] }
 
-axum = { version = "0.8.0-rc.1", features = [], optional = true }
+axum = { version = "0.8.1", features = [], optional = true }
 
 hyper-tls = { version = "0.6.0", optional = true }
-hyper-rustls = { version = "0.27.5", optional = true }
+hyper-rustls = { version = "0.27.5", optional = true, default-features = false, features = [
+    "http1",
+    "logging",
+    "native-tokio",
```

Comment:
Unsure if we want to do that, as the project itself does not depend on `tokio` directly.

Reply:
I just mirrored what default features are in hyper-rustls here after turning off default-features.  This is to exclude `aws-lc-rs` as not everyone is using that and it forces a default provider if it's included. 

IOW, this was there before in the default features. I don't mind excluding it if you prefer.